docs/settings.rst

```diff
@@ -984,6 +984,17 @@ Not recommended to set this below 3.
 
 Script to be used to edit incoming AXFRs, see :ref:`modes-of-operation-axfrfilter`
 
+.. _setting-lua-global-include-path:
+
+``lua-global-include-path``
+---------------------------
+
+-  String
+-  Default: empty
+-  Example: ``/etc/pdns/lua-global/*.lua``
+
+Path mask to include files to Lua contexts.
```

Comment:
So this loads all the files matching by glob? Every time?

Reply:
Yes, when the context is created. If the lua context exists, they are not loaded.

Reply:
I think it wouldn't hurt with a couple more words here.

Reply:
Better?